### PR TITLE
Add getting started blog

### PR DESCRIPTION
**Pull request by lauravuo**

### CODE REVIEW

File: content/en/blog/getting-started/index.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+```
+
+### CLI Script Issuing Credential
+
+The next task is to create a pairwise connection between the agent operated by the CLI user
+and the web wallet user. The pairwise connection is an encrypted pipe between the two
+agents that they can use to exchange data securely. The CLI script creates an invitation (**1-2**)
+and prints it out (**3**) as a QR code that the web wallet user can read (**5**).
+
+Once the new connection id is known, the CLI script starts a chatbot (**4**) for the new connection.
```
Comment (lainio) [inline content/en/blog/getting-started/index.md:97]: id -> ID

File: content/en/blog/getting-started/index.md [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+The bot logic follows the rules for changing the bot states in the YAML configuration file. Therefore,
```
Comment (lainio) [inline content/en/blog/getting-started/index.md:98]: YAML configuration file -> YAML file?
or YAML state machine file. The exact syntax will be published as a blog post soon.

Or something similar...

File: content/en/blog/getting-started/index.md [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+    Web Wallet->>Agency: Accept
+    Agency-->>Issue Bot: Issue ready!
+    Issue Bot->>Issue Bot: Terminate
+```
+
+### CLI Script Verifying Credential
+
+Steps **1-6** proceed similarly to the issuing: first, the agents form a new pairwise connection.
+However, the process continues with a proof request sent by the verifier bot (**7**). The proof request
+contains the attributes the bot wishes the holder to reveal. The web wallet user sees
```
Comment (lainio) [inline content/en/blog/getting-started/index.md:129]: reveal -> include in a proof presentation. 

Yes, this is hairsplitting, but if we want to keep ZKP in the game, that's more correct than 'revealing' the attributes, even when that's precisely what we are currently doing in most cases. We aren't using predicates in proof requests, etc.

File: content/en/blog/getting-started/index.md [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+    Verify Bot->>Verify Bot: Terminate
+```
+
+### CLI as a Test Tool
+
+Note that you can also utilize the CLI for testing.
+It is an excellent tool to simulate the functionality on the other end.
+
+For instance, let’s say you are developing an issuer service.
+You can use the CLI tool to manage the holder client and to receive the credential.
```
Comment (lainio) [inline content/en/blog/getting-started/index.md:170]: manage the holder -> play/act/simulate the holder

File: content/en/blog/getting-started/index.md [hunk 5]
```diff
@@ -0,0 +1,252 @@
...
+the requested data once they receive the message (**8**), and they can either accept or reject the request.
+
+Once the proof is accepted (**9**), the agency verifies it cryptographically.
+If the verification succeeds, the agency notifies the verifier bot with the proof values (**10**).
+It can reject the proof if the values are not acceptable by the business logic.
+The sample bot accepts all attribute values, so the verifying process is continued
+without extra validation (**11**).
+The bot exits when the proof is completed (**12-13**).
+
+And that’s all core features of verified data flow executed utilizing the CLI tool!
```
Comment (lainio) [inline content/en/blog/getting-started/index.md:139]: Just a comment to explain my next comment: here, you start to feel that we have reached the end. I mean as a reader.

Comment (lainio) [reply content/en/blog/getting-started/index.md:139]: ^ I mean my comment about splitting. Next level 2 heading.

File: content/en/blog/getting-started/index.md [hunk 6]
```diff
@@ -0,0 +1,252 @@
...
+It is an excellent tool to simulate the functionality on the other end.
+
+For instance, let’s say you are developing an issuer service.
+You can use the CLI tool to manage the holder client and to receive the credential.
+Or you can use the web wallet to hold the credential
+and create another client with the CLI tool to verify the issued data.
+
+## Run the Server Samples
+
+Ok, so you have now taken over the CLI tool, but how about the integration into your application?
```
Comment (lainio) [inline content/en/blog/getting-started/index.md:176]: Here we could start a new blog post that would also bring e.g. Go code blocks to the post to explain some key parts of our gRPC API. 

.. I would hate this comment as an author, more things to do. Writing and drawing sequence diagrams, ...

Still I think it's so close :-D

**Review by lainio — APPROVED** [review]
I loved this post: the rhythm, diagrams, language flow, everything.

As a reader, I felt here (the point where the first post would end) is an opportunity to switch the angle to coding aspects, and still, the first post is very needed because it's something everyone new to the stuff has to read and play with.

File: content/en/blog/getting-started/index.md [hunk 7]
```diff
@@ -0,0 +1,252 @@
...
+a complete set of tools for managing your digital wallet and agent via a user interface or an API.
+
+{{< imgproc cover Fit "825x825" >}}
+<em>Findy Agency provides tools for playing each role in the trust triangle: CLI and API clients
+have the complete tool pack, and the web wallet user can currently hold and prove credentials.
+</em>
+{{< /imgproc >}}
+
+The Agency tooling provides you with [a web wallet](https://github.com/findy-network/findy-wallet-pwa)
+and a CLI tool that you can use to test
```
[the remark this answers is not in the document]
Comment (lauravuo) [reply content/en/blog/getting-started/index.md:32]: Good idea.

File: content/en/blog/getting-started/index.md (hunk 3 above)
Comment (lauravuo) [reply content/en/blog/getting-started/index.md:129]: To present

File: content/en/blog/getting-started/index.md (hunk 4 above)
Comment (lauravuo) [reply content/en/blog/getting-started/index.md:170]: Act as the holder client

**Pull request merged**